hw/hw_0001/index.1.js

```diff
@@ -1,21 +1,26 @@
 let persons = [
-    {name: "Volodymyr", yearsOfBirth: 1981},
-    {name: "Elena", yearsOfBirth: 1988},
-    {name: "Steshka", yearsOfBirth: 2013},
-    {name: "Emilie", yearsOfBirth: 2015},
+  {name: "Volodymyr", surname: "Kravchenko" , yearsOfBirth: 1981},
```

Comment:
Please, rename name -> firstName surname -> lastName

Reply:
Ok i changed `name` and `surname` to suggested